recipes/nolearn/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "nolearn" %}
+{% set version = "0.6.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nolearn-{{ version }}.tar.gz
+  sha256: bab2dc2aeb9560264791298f1d206b78fd9b345cecdd500b34f469d367765158
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - joblib
+    - lasagne
+    - python >=3.6
+    - scikit-learn
```

So it looks like `scikit.cross_validation` was renamed, so should be able to get away with just a bound here:
```suggestion
    - scikit-learn <0.20
```